cnetstat.go

```diff
@@ -227,20 +191,15 @@ func cnetstat() error {
 
 	kubeConnections := getKubeConnections(allConnections, pidMap)
 
+	table := make([]Fielder, len(kubeConnections))
+	for i, _ := range kubeConnections {
+		table[i] = &kubeConnections[i]
+	}
+
 	switch format {
 	case "json":
-		for _, conn := range kubeConnections {
-			err := writeKubeConnectionAsJSON(&conn, os.Stdout)
-			if err != nil {
-				return err
-			}
-			os.Stdout.WriteString("\n")
-		}
+		printJsonTable(table, kubeConnectionHeaders, os.Stdout)
```

Comment:
This appears to be using the headers as the json field names. This is a change in behavior. Just making sure it is intentional.

Reply:
Yes, it's intentional. Thanks for checking! I wanted to have the same header names as field names, but I probably should have renamed the variable.